pkg/http/api_directmessaging.go

```diff
@@ -41,10 +41,9 @@ func (a *api) constructDirectMessagingEndpoints() []Endpoint {
 			Methods: []string{},
 			Route:   "invoke/*",
 			// This is the fallback route for when no other method is matched by the router
-			IsFallback:            true,
-			Version:               apiVersionV1,
-			KeepWildcardUnescaped: true,
```

Comment:
is true the only value for this bool in all code paths??

Reply:
KeepWildcardUnescaped was already a no-op since #6750 and it's "false" for all routes